freqtrade/freqai/data_kitchen.py

```diff
@@ -1126,13 +1126,57 @@ def set_all_pairs(self) -> None:
             if pair not in self.all_pairs:
                 self.all_pairs.append(pair)
 
+    def extract_corr_pair_columns_from_populated_indicators(
+        self,
+        dataframe: DataFrame
+    ) -> Dict[str, DataFrame]:
+        """
+        Find the columns of the dataframe corresponding to the corr_pairlist, save them
+        in a dictionary to be reused and attached to other pairs.
+        :params:
+        :dataframe: fully populated dataframe (current pair + corr_pairs)
+        :return:
+        :corr_dataframes: dictionary of dataframes to be attached to other pairs in same candle.
+        """
+        corr_dataframes: Dict[str, DataFrame] = {}
+        pairs = self.freqai_config["feature_parameters"].get("include_corr_pairlist", [])
+
+        for pair in pairs:
+            coin = pair.split('/')[0]
+            pair_cols = [col for col in dataframe.columns if coin in col]
```

Comment:
what'd be the side-effect if this matches a wrong column (which seems very likely ...)?

Based on the sample in the other comment - if my informative coin is `T/USDT` ([available on binance](https://www.binance.com/en/trade/T_USDT?)) - we'd have every column that contains a capital T (could be `BTC` - could be an indicator that i decided to name `-TRIPPLE` - and therefore would match on every column)

Reply:
See other comment for explanation on the new string checking method.